### PR TITLE
Intl.NumberFormat(): narrowSymbol for currencyDisplay option doesn't work in any browser

### PR DESCRIPTION
**Pull request by davidekete**
Removed the `narrowSymbol` option for `currencyDisplay` ("narrowSymbol": use a narrow format symbol ("$100" rather than "US$100"). It currently doesn't work, and the currency display falls back to `symbol`.

### Description

Removed the `narrowSymbol` option for `currencyDisplay` ("narrowSymbol": use a narrow format symbol ("$100" rather than "US$100"). It currently doesn't work, and the currency display falls back to `symbol`.

### Motivation
It is misleading since the option does not work.

### CODE REVIEW

**Comment by teoli2003** [comment]
On which browser did you test?

**Comment by davidekete** [comment]
> 

Chrome

**Comment by davidekete** [comment]
> 

Chrome

**Comment by teoli2003** [comment]
This is not working in Firefox, Safari, and Chrome. I agree we should remove it for the moment; also, we should update the note in BCD: Safari's note tells us of this limitation, but this note should be added to the other entries, too.

@Josh-Cena What do you think?

**Comment by Josh-Cena** [comment]
In the context of `Intl`, there's a very clear definition of what's supported and what's not: any option not supported will be outright rejected.

```js
new Intl.NumberFormat("en-US", {
  style: "currency",
  currency: "USD",
  currencyDisplay: "nope",
});
// RangeError: Value nope out of range for Intl.NumberFormat options property currencyDisplay
```

On the other hand, whether an option does anything meaningful at all is very tricky. There has never been any normative requirement on what browsers _must_ support for an option, so they can be spec-compliant by merely validating the options object, ignoring all fields, and formatting everything as if it's `en-US`.

Furthermore, it would be really hard to test browser support for an option. For example, what if, hypothetically, only the `bn-BD` locale supports the `narrowSymbol` option, but not anything else? What if they add support for one locale per major version release? You can't assume that `en-US` always has the most complete support.

In conclusion, I definitely think we should not remove it from the content, and I'm not sure if it's feasible to add any note to BCD.

**Comment by teoli2003** [comment]
Makes sense.

**Review by Josh-Cena — COMMENTED** [review]
In the context of `Intl`, there's a very clear definition of what's supported and what's not: any option not supported will be outright rejected.

```js
new Intl.NumberFormat("en-US", {
  style: "currency",
  currency: "USD",
  currencyDisplay: "nope",
});
// RangeError: Value nope out of range for Intl.NumberFormat options property currencyDisplay
```

On the other hand, whether an option does anything meaningful at all is very tricky. There has never been any normative requirement on what browsers _must_ support for an option, so they can be spec-compliant by merely validating the options object, ignoring all fields, and formatting everything as if it's `en-US`.

Furthermore, it would be really hard to test browser support for an option. For example, what if, hypothetically, only the `bn-BD` locale supports the `narrowSymbol` option, but not anything else? What if browsers add support for one locale per major version release? At which time can we consider it supported? You can't assume that `en-US` always has the most complete support (the example here is only for illustration of what should happen "in the best case").

In conclusion, I definitely think we should not remove it from the content, and I'm not sure if it's feasible to add any note to BCD.
